openedx_events/content_authoring/signals.py

```diff
@@ -14,6 +14,7 @@
     DuplicatedXBlockData,
     XBlockData,
 )
+from openedx_events.learning.data import CourseData
```

Comment:
@pomegranited Do you think that is code smell to import this data class from another subpackage? Is it better to create another data class?

Reply:
Hmm.. you're right, importing this from learning isn't good.. it specifically violates the [How-to for creating new events](https://github.com/openedx/openedx-events/blob/main/docs/how-tos/creating-new-events.rst#3-create-the-data-attributes-for-the-event-oep-49) (which I just found, while searching for the answer to your question here!):

> It should be created in the data.py file, as described in the OEP-49, in the corresponding architectural subdomain.

Also, we might want different data sent with these events than what gets sent for learning events.

So please create a new `openedx_events.content_authoring.data.CourseData` class. For now, it's ok to just include the `usage_key` there -- unless there's anything else available at that time which you think might be useful to us?